src/main/java/com/meraphorce/utils/ValidatorRequired.java

```diff
@@ -0,0 +1,16 @@
+package com.meraphorce.utils;
+
+import com.meraphorce.models.User;
+
+import java.util.Objects;
+
+public class ValidatorRequired {
+
+    public static void validateUser(Object user) {
+        // TODO implement a best validation
+
+        if (Objects.isNull(user)) {
+            throw new IllegalArgumentException("[ User is required ] ");
```

Comment:
**Small observation:** Since you already have a GlobalExceptionHandler, you could create a CustomException for this case.